docs/tutorials/autogluon-cloud.md

```diff
@@ -1,2 +1,119 @@
-# AutoGluon Cloud
-Coming Soon...
+# Deploying AutoGluon Models with AutoGluon Cloud on AWS SageMaker
+:label:`autogluon-cloud`
+
+To help with AutoGluon models training, AWS developed a set of training and inference [deep learning containers](https://github.com/aws/deep-learning-containers/blob/master/available_images.md#autogluon-training-containers). 
+The containers can be used to train models with CPU and GPU instances and deployed as a SageMaker endpoint or used as a batch transform job.
+
+We provide a full end-to-end example in [amazon-sagemaker-examples](https://github.com/aws/amazon-sagemaker-examples/tree/master/advanced_functionality/autogluon-tabular-containers) repository.
+
+In addition, we now offer the [autogluon.cloud](https://github.com/autogluon/autogluon-cloud) module to wrap functionality you saw in the example above with simple APIs.
+
+## Pre-requisites
+`autogluon.cloud` does not come with the default `autogluon` installation. Therefore, please install it via (`pip3 install autogluon.cloud`)
+Also ensure that the latest version of sagemaker python API is installed via (`pip3 install --upgrade sagemaker`). 
+This is required to ensure the information about newly released containers is available.
+
+## Prepare an AWS Role with Necessary Permissions
+`autogluon.cloud` utilizes various AWS resources to operate.
+To help you to setup the necessary permissions, you can generate trust relationship and iam policy with our utils through
+
+```{.python}
+from autogluon.cloud import TabularCloudPredictor  # Can be other CloudPredictor as well
+
+TabularCloudPredictor.generate_trust_relationship_and_iam_policy_file(
+    account_id="YOUR_ACCOUNT_ID",  # The AWS account ID you plan to use for CloudPredictor.
+    cloud_output_bucket="S3_BUCKET"  # S3 bucket name where intermediate artifacts will be uploaded and trained models should be saved. You need to create this bucket beforehand.
+)
+```
+
+The util function above would give you two json files describing the trust replationship and the iam policy.
+**Make sure you review those files and make necessary changes according to your use case before applying them.**
+
+## Training
+Using `autogluon.cloud` to train AutoGluon backed models are simple and not too much different from training an AutoGluon predictor directly.
+
+Currently, `autogluon.cloud` supports training/deploying `tabular`, `multimodal`, `text`, and `image` predictors. In the example below, we use `TabularCloudPredictor` for demonstration. You can substitute it with other `CloudPredictors` easily as they share the same APIs.
+
+```{.python}
+from autogluon.cloud import TabularCloudPredictor
+train_data = 'train.csv'
+predictor_init_args = {label='label'}  # init args you would pass to AG TabularPredictor
+predictor_fit_args = {train_data, time_limit=120}  # fit args you would pass to AG TabularPredictor
+cloud_predictor = TabularCloudPredictor(
+    cloud_output_path='YOUR_S3_BUCKET_PATH'
+).fit(
```

Comment:
Should provide option that automatically unattaches, so that the user doesn't need to manually exit if they want to do other things while the cloud predictor is training.

Reply:
User can just set `wait` to be false. I'll mention that